lib/to_openstudio/construction_set.rb

```diff
@@ -140,6 +140,8 @@ def to_openstudio(openstudio_model)
             @hash[:aperture_set][:window_construction])
           unless window_ref.empty?
             window_aperture = window_ref.get
+            #TODO: This looks wrong, it should be set to interior subsurface construction since
+            #window_construction apertures have a surface boundary condition.
```

Comment:
I think we can remove this now that we realized that this was just the result of a bug in honeybee-schema.